scripts/docker/content/set_env.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+
+AUTH_SERVER_URL=""
+RESOURCE_TYPE=""
+AUTH_TOKEN=""
+
+# Acquires auth token for authroizating against key vault.
+_acquire_token() {
```

Comment:
Can we use different naming conventions between method and variable names.

Reply:
This convention is used to define inner "private" functions. I just read about the conventions for bash scripts. If we have our own in this regard, I will change it to that.